.github/workflows/standard_library_tests_and_examples.yml

```diff
@@ -76,48 +76,14 @@ jobs:
 
 
           python3 -m pip install lit
-
-      - name: Filter files changed
-        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36
-        id: filter
-        with:
-          # Enable listing of files matching each filter.
-          # Paths to files will be available in `${FILTER_NAME}_files` output variable.
-          # Paths will be escaped and space-delimited.
-          # Output is usable as command-line argument list in Linux shell.
-          list-files: shell
-
-          filters: |
-            mojo:
-              - added|modified: '**/*.mojo'
-              - added|modified: '**/*.🔥'
-
-      # While this may seem like debugging, it's useful to see what's going on
-      # at a glance in this workflow.
-      - name: Print file changes
-        run: |
-          echo "Mojo files changed below:"
-          echo ${{ steps.filter.outputs.mojo_files }}
-
-      - name: mojo-formatting
-        if: ${{ steps.filter.outputs.mojo == 'true' }}
+      
+      - name: Install pre-commit
         run: |
-          # Make sure `mojo` is in the PATH.
-          # TODO: Refactor after https://github.com/modularml/mojo/pull/2032 lands
-          export PATH="/home/runner/.modular/pkg/packages.modular.com_nightly_mojo/bin:$PATH"
-          # The list of files from the filter step is relative to the checkout
-          # root.
-          cd $GITHUB_WORKSPACE
-          mojo format ${{ steps.filter.outputs.mojo_files }} -l 80
-          # Check if any lines were formatted.  If any were, fail this step.
-          if [ $(git diff | wc -l) -gt 0 ]; then
-            echo -e "\nError! Mojo code not formatted. Run `mojo format -l 80` ...\n"
-            echo -e "\nFiles that don't match the proper formatting:\n"
-            git diff --name-only
-            echo -e "\nFull diff:\n"
-            git diff
-            exit 1
-          fi
+          pip install pre-commit
+          pre-commit install
+      
+      - name: Run pre-commit
+        run: pre-commit run --all-files
```

Comment:
```suggestion
      - run: pre-commit run --all-files
```

Reply:
Can you expand on why you would want this change?

Reply:
Sure! I think it's just simpler and thus easier to maintain.

Reply:
Usually `name` can provide some sort of documentation for a series of complex steps, but here, there's only 1 step, and that step says almost the same thing as `name`, so this documentation seems unnecessary.

Reply:
Thank you for the opinion. I believe having steps in plain english helps the readability for people who are no familiar with the CI process. So I'd like to keep it. If the mainainers ask me to remove it, I'll comply though.

Reply:
You're welcome! That's fair.

Reply:
You're welcome! That's fair.